### PR TITLE
Subscription plan Modify UI snapshot

### PR DESCRIPTION
**Pull request by darshan-clvt**
### Subscription Plan contain two major changes for Dev.-portal 
_this feature is only superadmin User_ 
New ui option Home Page dev-portal

### manage_plan 
this pug scription handles all related to SMP
### admin.js 
where script is render

### CODE REVIEW

File: src/ui/views/manage_plans.pug [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+        return $('<div>').text(text).html();
+    }
+    $(function () {
+        $('[data-toggle="tooltip"]').tooltip()
+      })
+    var popupMessage = document.querySelector('.popup-message');
+                if (popupMessage) {
+                        popupMessage.style.display = 'none';
+                    }
+    var storedA, storedB, storedC, storedD ;
```
Comment (saichrn7) [inline src/ui/views/manage_plans.pug]: rename variables 'storeA,B,C,D ' to meaningful one

File: src/ui/routes/admin.js [hunk 2]
```diff
@@ -207,14 +207,14 @@ function getAdmin(req, res, uri, callback) {
         callback);
 }
 
-function getSubscriptions(req, res, next) {
```
Comment (saichrn7) [inline src/ui/routes/admin.js:210]: Can you find out different places at which  'getSubscriptions'  is used ?

File: src/ui/routes/admin.js [hunk 3]
```diff
@@ -347,7 +347,6 @@ router.get('/auditlog_csv', mustBeAdminOrApproverMiddleware, function (req, res,
             outStream.write('Api;Application;Plan;Date (UTC);Activity;User;Email;Role\n');
             for (let i = 0; i < auditResponse.items.length; ++i) {
                 const item = auditResponse.items[i];
-                utils.processDisplayNames(item);
```
Comment (saichrn7) [inline src/ui/routes/admin.js]: is this audit log related ? why are u removing it ?

File: src/ui/routes/admin.js [hunk 4]
```diff
@@ -692,4 +696,91 @@ router.post('/restart', mustBeAdminMiddleware, function (req, res, next) {
...
+                    const eachRequiredGroup = eachAllPlan.requiredGroup;
+                    let matchingUserGroup = userGroups.find(userGroup => userGroup === eachRequiredGroup);
+        
+                    if (matchingUserGroup) {
+                        requiredGroupValues.push(matchingUserGroup); 
+                    }
+                }
+            }
+            if (requiredGroupValues.length > 0){
+                finalResult = allowRefinedPlan.map(x => {
```
Comment (saichrn7) [inline src/ui/routes/admin.js]: instead x, can you use valid variable name ?

**Review by saichrn7 — CHANGES_REQUESTED** [review]

File: src/ui/routes/admin.js (hunk 2 above)
Comment (darshan-clvt) [reply src/ui/routes/admin.js:210]: function is called in two place for **admin subscriptions** routes and **mangeplans** routes only, otherwise it not used any where.

File: src/ui/routes/admin.js (hunk 3 above)
Comment (darshan-clvt) [reply src/ui/routes/admin.js]: Sure will add It again
